### PR TITLE
Hotfix internal links and refs

### PR DESCRIPTION
**Pull request by spirillen**
This is a hotfix, to solve the issues sphinx internal links and refs

It isn't a pretty fork, but it should solve the issue ASAP. It will needs more love.

### CODE REVIEW

File: docs/installation/requirements.rst [hunk 1]
```diff
@@ -10,7 +10,7 @@ Here is the list of requirements:
 -   :code:`dnspython`
 -   :code:`domain2idna`
 -   :code:`inflection`
--   :code:`PyMySQL`like - for example - Travis CI
+-   :code:`PyMySQL` like - for example - Travis CI
```
Comment (funilrys) [inline docs/installation/requirements.rst:13]: Uhh can you please delete the `like - for example - Travis CI`. I can't remember how it ended there 🤔

File: docs/usage/from-a-terminal.rst [hunk 2]
```diff
@@ -1130,7 +1117,7 @@ Global overview
         -h, --host            Activates or disables the generation of the
                                 hosts file(s).
                                 Configured value: True
-        -ip CLI_TESTING__HOSTS_IP, --hosts-ip CLI_TESTING__HOSTS_IP
+        -ip CLI_TESTING__HOSTS_IP, --host-ip CLI_TESTING__HOSTS_IP
```
Comment (funilrys) [inline docs/usage/from-a-terminal.rst:1120]: It is called `hosts` file 😉 

But I'm open to adding the singular version. Can you please add/modify it in the resource code too ? 

https://github.com/funilrys/PyFunceble/blob/2b12f88e69d549e52de681c1fd490c6f4c54ec04/PyFunceble/cli/entry_points/pyfunceble/cli.py#L647

**Review by funilrys — CHANGES_REQUESTED** [review]
Still, some little things and it will hopefully good for merging.

Thank you for your time!

File: docs/installation/requirements.rst (hunk 1 above)
Comment (spirillen) [reply docs/installation/requirements.rst:13]: Sure thing, I was kind a wondering the same thing.

File: docs/usage/from-a-terminal.rst (hunk 2 above)
Comment (spirillen) [reply docs/usage/from-a-terminal.rst:1121]: It was and find/replace error

Should be fixed

**Review by funilrys — APPROVED** [review]
Awesome thank you for your time!

**Comment by spirillen** [comment]
And thanks for your time :wave: You are pulling the biggest work with all the code, I'm just fiddling with the docs

**Pull request merged**